### DISCUSSION TITLE
How to get standard structures and variables from a gcc library file in your wrapper file

### DISCUSSION THREAD

**Discussion by Wildfire2110** [swig/swig#2823] [Q&A]
I am using SWIG library for one of the project where I the target language is python and bases is .c files 
I have a standard library files from gcc library
How can I access the variables from that library

**Comment by gogo2464** [comment]
I am confused. Do you mean you would like to do `swig -python process.i` to have the wrapper?

**Reply by gogo2464** [reply]
@Wildfire2110 your question is very unclear. Did you checked my example here: https://github.com/gogo2464/pwnto-driver/blob/main/pwnlib/tubes/Makefile#L7 ? :

```
	swig -python process.i
	gcc -O2 -fPIC -c process.c
	gcc -O2 -fPIC -c process_wrap.c -I"$(PYENV)\include"
	gcc -shared process.o process_wrap.o -o _process.pyd -L "$(PYENV)\libs" -l python3
```

**Comment by gogo2464** [comment]
does it answer to your question?

**Comment by ojwb** [comment]
No response so assuming answered (I have no clear idea what was being asked here).
